src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -588,7 +588,96 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
+
+The Azure Scale Set Infrastructure offers a wide range of possibilities to configure your Scale Set so that:
+[lowerroman]
+. You have in your VMs everything you need to execute your jobs.
+. You can control your expenses by defining size and limits of your VMs.
+
+The available configuration parameters are:
+
+- *azureCredentialFile:* Use the provided file selection tool to supply a file with your Azure Credentials. These credentials correspond to those generated with the Service Principal as mentioned in the Pre-Requisites sub-section. The file should be a four-line text document in a `.ini` like format, and must include the following elements:
++
+....
+client=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+key=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+tenant=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+subscription=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+....
+Where:
++
+* *_client_:* Corresponds to the Service Principal _application ID_.
+* *_key_:* Corresponds to the _secret_ value (password) of the Service Principal.
+* *_tenant_:* Corresponds to the Azure _domain_, _tenant ID_, or _directory ID_ of your Active Directory.
+* *_subscription_:* Is the _ID_ of your Azure subscription.
+
++
+You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.creds` so that it will available and automatically loaded in case you want to have multiple Azure node sources.
+
+- *maxVms:* Define the maximum number of VMs to be deployed by the scale set at any time. The number cannot exceed 100. The default value is _100_, so you might want to adjust this value according to your budget and expected processing peaks.
+- *maxNodesPerVm:* Define the number of ProActive worker nodes to be deployed per VM. As a guideline, you might want to allocate one node per VM core, but you can set the number to better match your processing requirements. Default value is _2_.
+- *machineType:* Define the type of Azure VM that you want to use for your Scale Set. You can use any size from Azure's Standard Tier, provided you have enough quota and budget to launch them. The default value is _Standard_D1_v2_, you should provide a valid type using a similar syntax (case insensitive). Azure provides an extensive list of VM sizes for https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes[Linux] and https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes[Windows] VMs. If you want to know the available sizes for a specific region or subscription you can use https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-list-sizes[Azure CLI command] `az vm list-sizes -l {your location}`.
+- *osType:* A choice between _linux_ and _windows_. Default value is _linux_.
+- *Image:* You can either provide a custom Azure image, or select one of the available standard ones. To provide a custom image use the format _{resourceGroupName}:{imageName}_. If you prefer to use a generic image, you can choose from the following lists (provided by Azure API):
```

Comment:
A pointer to the Azure documentation could be a great complement for this section.

Reply:
Done, I added the links on the table headers.